buildSrc/src/main/kotlin/splunk.java-conventions.gradle.kts

```diff
@@ -13,11 +13,11 @@ repositories {
   }
 }
 
-val otelVersion = "1.10.0"
-val otelAlphaVersion = "1.10.0-alpha"
+val otelVersion = "1.10.1"
```

Comment:
there is also the hard to remember `implementation("io.opentelemetry.instrumentation:gradle-plugins:1.10.0-alpha")` in `buildSrc/build.gradle.kts`